Gems/Multiplayer/Code/Source/Components/NetBindComponent.cpp

```diff
@@ -212,6 +212,16 @@ namespace Multiplayer
         return (m_netEntityRole == NetEntityRole::Client);
     }
 
+    void NetBindComponent::SetAllowEntityMigration(EntityMigration value)
+    {
+        m_netEntityMigration = value;
+    }
+
+    EntityMigration NetBindComponent::GetAllowEntityMigration() const
+    {
+        return m_netEntityMigration;
+    }
```

Comment:
This relates to the ability to migrate for entity that owns this component instance right? SetCanEntityMigrate/CanEntityMigrate might read a bit more clearly. I'd also be very clear about how this works with hierarchies in comments.